### DISCUSSION TITLE
PicoTorrent discussions!

### DISCUSSION THREAD

**Discussion by vktr** [picotorrent/picotorrent#1054] [General]
Hey all! 👋 

We've finally got access to GitHub Discussions which is awesome! I'm thinking this is the perfect place to have discussions about upcoming features, missing features, general ideas and so forth!

We have a [Discord server](https://discord.gg/tV3dFrv) but not everyone is on Discord, so I think these two will complement each other very well!

Looking forward to discussing anything 😄

**Comment by make-42** [comment]
the 🌶️ culmination.

**Comment by alex19EP** [comment]
yay!

**Comment by mitchger** [comment]
I love PicoTorrent! I just really miss auto-loading of torrent files from my download folder that I enjoyed with others. Any chance you'd consider adding this feature?

**Reply by vktr** [reply]
Hi! Sorry for the late reply - yes, monitored folders is something I've been meaning to add! I just haven't had the time/energy lately.

**Comment by ghost** [comment]
Hi all!
